docs/docs/overview/supported-tests.mdx

```diff
@@ -0,0 +1,76 @@
+import DocCard from "../../src/theme/DocCard";
+import clsx from "clsx";
+
+# Supported Test Types
+
+Testkube supports running a long list of testing tools and frameworks. You can find the list of supported test types below.
+
+If we're not currently supporting a testing framework you need, you can create a [Custom Testkube Executor](../test-types/executor-custom.md) and configure it to run any type of tests that you want. These custom executors can be added to your Testkube installation and/or contributed to our repo.
```

Comment:
Almost noone wil do that - I think it's better to mention Container executors first

Reply:
great point